CONTRIBUTING.md

```diff
@@ -99,6 +99,13 @@ submit them to the base repository using a pull request.
     
     Where `<branch>` is the branch name you used in step 2.
 
+    If asked for github username and password, make sure to use your [Personal
```

Comment:
I forget--is 2FA a requirement for all contributors to google repos? Or is it just a requirement for googlers?

Either way looks good, just asking since if it only applies to googlers we may want to make this a note or something instead of weaving it into the main text.

Reply:
I believe you are right! In my security setting it says 2FA is required because your account is affiliated with an org or corporation that requires it.